app/src/main/java/com/example/whatsapp_clone/Model/Retrofit/HTTPClientDataSource.java

```diff
@@ -84,7 +125,7 @@ public void onFailure(@NonNull Call<List<Chat>> call, @NonNull Throwable t) {
     }
 
     public void createChat(String token, String username, CompletionBlock<Chat> response) {
-        
+
```

Comment:
make sure to pull from main and be up to date again, there is logic in here that is not implemented in your branch